addons/survey/views/survey_templates_statistics.xml

```diff
@@ -106,29 +106,31 @@
                     </t>
                 </div>
             </div>
-            <!-- Question info -->
-            <span class="badge badge-info" t-field='question.question_type'/>
-            <t t-if="question.question_type == 'matrix'">
-                <span class="badge badge-info ml-2" t-field='question.matrix_subtype'/>
-            </t>
-            <!-- Scoring info -->
-            <t t-if="question_scoring">
-                <t t-if="question.question_type in ['simple_choice', 'multiple_choice'] or question.is_scored_question">
-                    <span class="badge badge-success ml-3"><span t-esc="question_data['right_inputs_count']"></span> Correct</span>
+            <div class="ml-auto">
```

Comment:
Hello @adsh-odoo,
Our stable policy prevents us from making this particular change unless absolutely necessary because of a crash, mainly because changing the xml arch/DOM can lead to breaking other overrides (community code).

Is it possible to avoid it?